innative/compile.h

```diff
@@ -110,7 +113,7 @@ namespace innative {
     void PolymorphicStack();
     llvmVal* GetMemSize(llvm::GlobalVariable* target);
     varuint32 GetFirstType(varuint32 type);
-    llvmVal* GetMemPointer(llvmVal* base, llvm::PointerType* pointer_type, varuint7 memory, varuint32 offset);
+    llvmVal* GetMemPointer(llvmVal* base, llvm::PointerType* pointer_type, varuint32 memory, varuint32 offset);
```

Comment:
As stated above, please double check that memory indices were actually changed to 32-bit integers.